esphome/__main__.py

```diff
@@ -681,6 +750,12 @@ def parse_args(argv):
         "configuration", help="Your YAML configuration file(s).", nargs=1
     )
 
+    parser_rename = subparsers.add_parser("rename")
+    parser_rename.add_argument(
+        "configuration", help="Your YAML configuration file(s).", nargs=1
```

Comment:
Since we only accept one. 
```suggestion
        "configuration", help="Your YAML configuration file.", nargs=1
```